spec/src/main/asciidoc/configprofile.asciidoc

```diff
@@ -75,6 +74,18 @@ In more details, if `mp.config.profile` is set to `dev`, the property `%dev.vehi
 The properties `%live.vehicle.name` and `%testing.vehicle.name` are inactive config properties and don't override the property `vehicle.name`.
 
 If `mp.config.profile` is set to `live`, the property `%live.vehicle.name` is the active property. The `vehicleName` will be `train`. Similarly, `bike` will be the value of `vehicleName`, if the profile is `testing`.
+If no profile is active, `lorry` will be the value of `vehicleName`.
+
+When another configuration source provides the same property, the final resulting value of the lookup depends on the ordinal (priority) of all providing sources.
+
+For instance, an additional source (ordinal 300) may be specified as follows:
+
+[source, text]
+----
+vehicle.name=helicopter
+----
+
+The value of `vehicleName` will be `helicopter` no matter which profile might be active, as the higher ordinal (300 versus 100) makes the later source win.
```

Comment:
For the above sentences, I suggest to just say:
The config profile controls which property value will be surfaced beyond each configure source. In other words, the override scope is per configure source. You can then add an example to explain that if you like.

Reply:
Thanks, refactored.